src/gui.js

```diff
@@ -1255,6 +1256,27 @@ IDE_Morph.prototype.createCategories = function () {
     this.categories.bounds.setWidth(this.paletteWidth);
     // this.categories.getRenderColor = ScriptsMorph.prototype.getRenderColor;
 
+
+    if (this.scene.unifiedPalette) {
+        categorySelectionAction = scrollToCategory;
+    } else {
+        categorySelectionAction = changePallete;
```

Comment:
umm, not sure (I always have to look this up) but isn't it spelled "palette"?

Reply:
also, how did I start a "review", I just wanted to comment... haha

Reply:
Ooops! One L. :) at least autocomplete works! 

And haha, GitHub’s default thing is to leave a review now.